### PR TITLE
Create dropout.py

### PR DESCRIPTION
**Pull request by DevilLord9967**
Adding dropout Neural network Algorithm implementation

### CODE REVIEW

**Comment by cclauss** [comment]
https://travis-ci.com/TheAlgorithms/Python/builds/133435240#L333

**Comment by cclauss** [comment]
@yusugomori are your review of this submission please.

File: neural_network/dropout.py [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+def dReLU(x):
+    return 1. * (x > 0)
+
+
+
+'''
+Dropout
+'''
+class Dropout(object):
+    def __init__(self, input, label,\
```
Comment (cclauss) [inline neural_network/dropout.py]: Please remove all six backslashes from this file.  They are not needed and their use is discouraged in PEP8.

Comment (DevilLord9967) [reply neural_network/dropout.py]: okay

**Pull request closed without merging**